lib/manageiq/appliance_console/cli.rb

```diff
@@ -280,6 +286,15 @@ def set_time_zone
       end
     end
 
+    def set_date_time
+      date_time_config = ManageIQ::ApplianceConsole::DateTimeConfiguration.new
+      unless options[:datetime] == "auto"
+        date_time_config.manual_time_sync = true
+        date_time_config.new_date, date_time_config.new_time = options[:datetime].split("_")
```

Comment:
Can we do validation against the regular expressions defined in `DateTimeConfiguration`?

When used interactively those are applied during `#ask_questions` so they will not get applied here.